data/schema.graphql

```diff
@@ -2499,6 +2560,7 @@ type OrderedSet {
 
 type organizer {
   profile_id: ID
+  profile: Profile
```

Comment:
Yep ( [this object](https://github.com/artsy/metaphysics/blob/master/schema/profile.js) ) which gets attached to things